Dockerfile

```diff
@@ -4,20 +4,19 @@ MAINTAINER ManageIQ https://github.com/ManageIQ/manageiq
 
 ENV DATABASE_URL=postgresql://root@localhost/vmdb_production?encoding=utf8&pool=5&wait_timeout=5
 
-RUN dnf -y --setopt=tsflags=nodocs install \
+RUN echo "# This file intentionally left blank. ManageIQ maintains its own SSL configuration" > /etc/httpd/conf.d/ssl.conf && \
```

Comment:
I feel like this line should be a separate RUN command (since it's not installing anything), but I'm not going to hold up the PR.